src/main/java/io/jenkins/plugins/audit/listeners/SavableChangeListener.java

```diff
@@ -0,0 +1,37 @@
+package io.jenkins.plugins.audit.listeners;
+
+import hudson.Extension;
+import hudson.XmlFile;
+import hudson.model.Fingerprint;
+import hudson.model.FingerprintCleanupThread;
+import hudson.model.FingerprintMap;
+import hudson.model.Saveable;
+import hudson.model.listeners.SaveableListener;
+import hudson.tasks.Fingerprinter;
+import io.jenkins.plugins.audit.event.CredentialsUsage;
+import jenkins.model.FingerprintFacet;
+import org.apache.logging.log4j.audit.LogEventFactory;
+
+import javax.annotation.Nonnull;
+import java.util.Collection;
+import java.util.List;
+
+@Extension
+public class SavableChangeListener extends SaveableListener {
```

Comment:
I recommend renaming this to "SaveableChangeListener".

Reply:
Thanks @jeffret-b